main/url/team_urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from main.controller import team_controller
+
+
+urlpatterns = [
+    path('create', team_controller.CreateTeamView.as_view(), name='create_team'),
+    path('<int:team_id>', team_controller.get_team, name='get_team'),
```

Comment:
@sushantdey if we want to create team, the url as per REST standards will be simply :
POST /team
we don't need "create" string in the path.

To fetch all teams, the url would simply be:
GET /team

To fetch a team by id, url would simply be:
GET /team/<:id>